app/views/groups/index.html.erb

```diff
@@ -0,0 +1,11 @@
+<h1> All groups </h1>
```

Comment:
- [ ] In the mobile version of your app, the hamburger button does not work. I recommend fixing that so the user is able to navigate between pages